### ISSUE TITLE
JOSS Paper Submission 

### ISSUE THREAD

**Issue by tkoyama010** [bjlittle/geovista#389]
## 📚 Documentation

At SciPy2023, we found that many researchers needed GeoVista. We propose to submit this project to JOSS so that researchers will have no trouble using GeoVista as a reference in the future.

[The Journal of Open Source Software](https://joss.theoj.org/) is a developer friendly, open access journal for research software packages.

**Comment by bjlittle** [comment]
@tkoyama010 Yup, this is high on my `todo` list. Thanks for the reminder :+1:

**Comment by dennissergeev** [comment]
I'd be keen to be help with this!

**Comment by banesullivan** [comment]
Let me know if I can play a supporting role here! I would love to see this submitted to JOSS!

**Comment by tkoyama010** [comment]
This is still important.
